src/patients/related-persons/RelatedPersonTab.tsx

```diff
@@ -87,14 +90,18 @@ const RelatedPersonTab = (props: Props) => {
       <div className="row">
         <div className="col-md-12">
           <Panel title={t('patient.relatedPersons.label')} color="primary" collapsible>
-            {relatedPersons ? (
-              <List>
-                {relatedPersons.map((r) => (
-                  <ListItem key={r.id}>{r.fullName}</ListItem>
-                ))}
-              </List>
-            ) : (
-              <h1>Loading...</h1>
+            {dataFetched ? (
+              relatedPersons ? (
+                <List>
+                  {relatedPersons.map((r) => (
+                    <ListItem key={r.id}>{r.fullName}</ListItem>
+                  ))}
+                </List>
+                  ) : (
+                <h1>No related persons have been added yet.</h1>
```

Comment:
What do you think about using the `Alert` component with the `warning` color?

Reply:
Yes. It's much better.